src/main/java/com/team4/testingsystem/dto/QuestionDTO.java

```diff
@@ -0,0 +1,91 @@
+package com.team4.testingsystem.dto;
+
+import com.team4.testingsystem.entities.Level;
+import com.team4.testingsystem.entities.Module;
+import com.team4.testingsystem.entities.User;
+
+public class QuestionDTO {
+    private String questionBody;
+    private boolean isAvailable;
+    private Long creatorId;
+    private Long levelId;
+    private Long moduleId;
+
+
+    public Long getCreatorId() {
+        return creatorId;
+    }
+
+    public void setCreatorId(Long creatorId) {
+        this.creatorId = creatorId;
+    }
+
+    public String getQuestionBody() {
+        return questionBody;
+    }
+
+    public void setQuestionBody(String questionBody) {
+        this.questionBody = questionBody;
+    }
+
+    public boolean isAvailable() {
+        return isAvailable;
+    }
+
+    public void setAvailable(boolean available) {
+        isAvailable = available;
+    }
+
+    public Long getLevelId() {
+        return levelId;
+    }
+
+    public void setLevelId(Long levelId) {
+        this.levelId = levelId;
+    }
+
+    public Long getModuleId() {
+        return moduleId;
+    }
+
+    public void setModuleId(Long moduleId) {
+        this.moduleId = moduleId;
+    }
+
+    public static class Builder {
+        private QuestionDTO questionDTO;
+
+        public Builder() {
+            this.questionDTO = new QuestionDTO();
+        }
+
+        public Builder questionBody(String questionBody) {
+            questionDTO.questionBody = questionBody;
+            return this;
+        }
+
+        public Builder isAvailable(boolean isAvailable) {
+            questionDTO.isAvailable = isAvailable;
+            return this;
+        }
+
+        public Builder creatorId(User user){
+            questionDTO.creatorId = user.getId();
+            return  this;
+        }
+
+        public Builder levelId(Level level) {
```

Comment:
Seams really confusing. LevelId method in builder expected to receive an Id of a level.

Alternatively, it could be a method level, which will save only id in dto. 

Anyway, builder method should define, what are they expected to receive